.github/workflows/dialyzer-regression.yml

```
@@ -0,0 +1,178 @@
+    - name: Check for regression
+      run: |
+        # Run the regression test
+        mix test test/dialyzer_regression_test.exs
+
+    - name: Comment PR with metrics
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const fs = require('fs');
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)